packages/plugins/rubygem-smart_proxy_discovery/rubygem-smart_proxy_discovery.spec

```diff
@@ -80,7 +96,9 @@ mv %{buildroot}%{gem_instdir}/settings.d/discovery.yml.example \
 %dir %{gem_instdir}
 %config(noreplace) %attr(0640, root, foreman-proxy) %{foreman_proxy_settingsd_dir}/discovery.yml
 %license %{gem_instdir}/LICENSE
+%{gem_instdir}/bundler.d
```

Comment:
This one is odd since we already have `%{foreman_proxy_bundlerd_dir}/%{plugin_name}.rb` below.